src/main/java/teammates/ui/controller/AdminActivityLogPageAction.java

```diff
@@ -118,6 +118,12 @@ private void generateStatusMessage(AdminActivityLogPageData data, List<ActivityL
         if (data.isPersonSpecified()) {
             String targetUserGoogleId = data.getPersonSpecified();
             targetTimeZone = getLocalTimeZoneForRequest(targetUserGoogleId, "");
+            // if cannot find the user or the user is unregistered
```

Comment:
I think this is an inaccurate comment, there are other reasons why the time zone will be uninitialised